### PR TITLE
Add header meta tags

### PR DESCRIPTION
**Pull request by pedromcunha**

### CODE REVIEW

File: pages/profile/[address].tsx [hunk 1]
```diff
@@ -133,6 +134,9 @@ const IndexPage: NextPage<Props> = ({ address, ssr, ensName }) => {
 
   return (
     <Layout>
+      {/* <Head>
+        <title>{`${address} Profile`}</title>
```
Comment (ted-palmer) [inline pages/profile/[address].tsx:138]: Can probably remove this comment

File: pages/collection/[chain]/[contract]/[id].tsx [hunk 2]
```diff
@@ -109,6 +110,22 @@ const IndexPage: NextPage<Props> = ({ id, collectionId, ssr }) => {
...
+            : `${token?.token?.tokenId} - ${token?.token?.collection}`}
+        </title>
+        <meta name="description" content={collection?.description as string} />
+        <meta
+          name="twitter:image"
+          content={token?.token?.image || collection?.banner}
+        />
+        <meta
+          property="og:image"
+          content={token?.token?.image || collection?.banner}
```
Comment (ted-palmer) [inline pages/collection/[chain]/[contract]/[id].tsx:126]: Do we want to add the token name / token id to the og title?

File: pages/collection/[chain]/[contract].tsx [hunk 3]
```diff
@@ -139,6 +140,21 @@ const CollectionPage: NextPage<Props> = ({ id, ssr }) => {
...
+          content={ssr?.collection?.collections?.[0]?.description as string}
+        />
+        <meta
+          name="twitter:image"
+          content={ssr?.collection?.collections?.[0]?.banner}
+        />
+        <meta
+          property="og:image"
+          content={ssr?.collection?.collections?.[0]?.banner}
+        />
```
Comment (ted-palmer) [inline pages/collection/[chain]/[contract].tsx:156]: Should we add the collection name to the og title?

File: pages/collection/[chain]/[contract]/[id].tsx (hunk 2 above)
Comment (pedromcunha) [reply pages/collection/[chain]/[contract]/[id].tsx:126]: good call. I'll update that

File: pages/profile/[address].tsx (hunk 1 above)
Comment (pedromcunha) [reply pages/profile/[address].tsx:138]: Forgot to uncomment this

**Review by ted-palmer — APPROVED** [review]
Looks good!

**Pull request merged**